packages/compartment-mapper/src/parse-cjs.js

```diff
@@ -25,35 +25,44 @@ export const parseCjs = async (
    * @param {Compartment} compartment
    * @param {Record<string, string>} resolvedImports
    */
-  const execute = async (moduleExports, compartment, resolvedImports) => {
```

Comment:
There doesn't seem to be a need for this to be async anywhere and we're risking unhandled rejections leaving it as-is. Removed async.